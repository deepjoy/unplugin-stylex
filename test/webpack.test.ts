import { expect, test } from 'vitest'
import stylexWebpackPlugin from '../src/webpack'

test('webpack should throw error', () => {
  expect(() => stylexWebpackPlugin()).toThrowErrorMatchingSnapshot()
  expect(() => stylexWebpackPlugin({ dev: true })).toThrowErrorMatchingSnapshot()
})
