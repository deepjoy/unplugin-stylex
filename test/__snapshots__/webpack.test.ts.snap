// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`throw error 1`] = `[Error: If you want to use this plugin through webpack, please use "@stylexjs/webpack-plugin" instead]`;

exports[`webpack should throw error 1`] = `[Error: If you want to use this plugin through webpack, please use "@stylexjs/webpack-plugin" instead]`;

exports[`webpack should throw error 2`] = `[Error: If you want to use this plugin through webpack, please use "@stylexjs/webpack-plugin" instead]`;
